xmodule/tasks.py

```diff
@@ -0,0 +1,317 @@
+"""
+This module contains tasks for asynchronous execution of library xblocks.
```

Comment:
Nit:
```suggestion
This module contains tasks for asynchronous updates of library xblocks.
```
or something like that... A library XBlock cannot be "executed" so this phrasing sounded weird.

Reply:
https://github.com/openedx/edx-platform/pull/33263#pullrequestreview-1627813331